Source/EventFlow/Sagas/DispatchToSagas.cs

```diff
@@ -83,6 +83,13 @@ public class DispatchToSagas : IDispatchToSagas
             {
                 var locator = (ISagaLocator) _resolver.Resolve(details.SagaLocatorType);
                 var sagaId = await locator.LocateSagaAsync(domainEvent, cancellationToken).ConfigureAwait(false);
+
+                if (sagaId == null)
+                {
+                    _log.Verbose(() => $"Saga locator '{details.SagaLocatorType.Name}' returned null");
```

Comment:
Instead of `'{details.SagaLocatorType.Name}'`, write `'{details.SagaLocatorType.PrettyPrint()}'` to get a more readable type name for logging